src/Xrefcheck/CLI.hs

```diff
@@ -150,4 +152,32 @@ getCommand = do
         info (helper <*> versionOption <*> totalParser) $
         fullDesc <>
         progDesc "Cross-references verifier for markdown documentation in \
-                 \Git repositories."
+                 \Git repositories." <>
+        (footerDoc $ pure ignoreModesMsg)
+
+ignoreModesMsg :: Doc
+ignoreModesMsg = text $ header <> body
+    where
+        header = "To ignore a link in your markdown, \
+                 \include \"<!-- xrefcheck: ignore mode -->\"\n\
```

Comment:
I would wrap "mode" into something, braces or angle brackets, to make it clear that only this word it to be replaced with actual mode.